app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <string name="AppName">Picover</string>
     <string name="Parties">Parties</string>
+    <string name="fighter">Kamrat</string>
```

Comment:
We decided to use PascalCase capitalization for string names!
![image](https://user-images.githubusercontent.com/29087969/234837376-843dea92-8634-46d9-a802-58dba5ee9526.png)